### DISCUSSION TITLE
Why isn't Perl supported?

### DISCUSSION THREAD

**Discussion by matthewpersico** [community/community#126434] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

There is plenty of support for parsing Perl. https://tree-sitter.github.io/tree-sitter/ has https://github.com/tree-sitter-perl/tree-sitter-perl and https://github.com/tree-sitter-perl/tree-sitter-pod listed. Can we get them integrated into GitHub?

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Comment by look** [comment, marked as answer]
@matthewpersico thanks for the suggestion. I'm working on documentation for how language communities can help us add support to their language for code navigation. I'll post an update here when it's been made public.

**Reply by matthewpersico** [reply]
Thanks! Looking forward to it.

**Reply by look** [reply]
@matthewpersico the documentation for supporting a new language has been released! Please check out https://github.com/github/code-navigation and review the process for submitting a new language.
